src/views/CurrentReservations.vue

```diff
@@ -12,23 +12,29 @@
     <b-button class="reservation-item"
     @click="releaseStreet(street.name + ' ' + street.type)" variant="danger">
       Release</b-button>
+    <b-button class="reservation-item" @click="() => toEditReservations(street.FID)">
+      Edit reservation</b-button>
   </b-row>
   </b-container>
 </div>
 </template>
 
 <script>
+import { mapState } from 'vuex';
+
 export default {
   name: 'CurrentReservations',
-  data: () => ({
-    streets: [
-      { name: 'Abraham', type: 'ST' },
-      { name: 'Hemenway', type: '' },
-      { name: 'Hunington', type: 'BLVD' },
-      { name: 'Parker', type: 'AVE' },
-    ],
-  }),
+  computed: {
+    ...mapState(['reservedStreets']),
+    reservedStreets() {
```

Comment:
Do lines 28 and 29 not duplicate each other? I might be reading this wrong. @bensenescu